cmd/osbuild-worker/jobimpl-depsolve.go

```diff
@@ -30,6 +32,16 @@ func (impl *DepsolveJobImpl) Run(job worker.Job) error {
 		return err
 	}
 
+	for _, r := range args.Repos {
```

Comment:
Why a loop?

I would even suggest making this dead simple and just creating a new RPPMD object for each job.

Reply:
well i thought only refresh in case the repos require rhsm